desktop/apps/editorial_features/components/venice_2017/client/index.coffee

```diff
@@ -45,6 +47,12 @@ module.exports = class VeniceView extends Backbone.View
     @VeniceVideoView.trigger 'swapVideo',
```

Comment:
Also note that in your .env you'll want to update the WHITELISTED_VANITY_ASSETS to be `videos/*|vrview/*` now that we're using regex matching instead of pulling out each individual file.